src/fiatconnect/TransferStatusScreen.tsx

```diff
@@ -63,22 +81,39 @@ function FiatConnectWithdrawFailureSection() {
 }
 
 function FiatConnectWithdrawSuccessSection({
+  flow,
   fiatConnectTransfer,
   normalizedQuote,
 }: {
+  flow: CICOFlow
   fiatConnectTransfer: FiatConnectTransfer
   normalizedQuote: FiatConnectQuote
 }) {
   const { t } = useTranslation()
   // TODO: Make sure there's fallback text if we can't get the estimate
   const timeEstimation = normalizedQuote.getTimeEstimation()!
+  const provider = normalizedQuote.getProviderId()
 
   const onPressTxDetails = () => {
+    ValoraAnalytics.track(FiatExchangeEvents.cico_fc_transfer_success_view_tx, {
+      flow,
+      provider,
+      txHash: fiatConnectTransfer.txHash!,
```

Comment:
`fiatConnectTransfer.txHash` has the type `string | null`, but `txHash` expects `string | undefined`, so we need a non-null assertion here (which is OK, since we're sure it exists here).